config/networking/istio/README.md

```diff
@@ -0,0 +1,9 @@
+## Istio configuration
```

Comment:
We feel that this should probably be a separate directory inside `config`, instead of a sub-directory of `networking`.

Reply:
I think that makes sense, we made this change just now :)